lib/logster/group.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+module Logster
+  class Group
+    MAX_SIZE = 100
+
+    attr_reader :key, :messages_keys, :timestamp, :messages
+    attr_accessor :changed
+
+    def initialize(key, messages_keys = [], timestamp: nil)
+      @key = key
+      @messages_keys = messages_keys || []
+      @timestamp = timestamp || 0
```

Comment:
I got small cosmetic suggestion here. <aybe it would be better to pass 0 as default to timestamp so we could be able to skip `|| 0`